pkg/resourceutils/updatereplicas.go

```diff
@@ -31,200 +31,113 @@ const (
 )
 
 func ScaleReplicas(namespace string, activate bool, printFunc func(string, string), config *rest.Config) {
```

Comment:
As part of this PR, I have restored updatereplicas file to master's version. 
We will not be using the existing scaling logic used by storkctl activate/deactivate.